tiledb/bioimg/converters/base.py

```diff
@@ -255,10 +273,12 @@ def tile_to_tiledb(level_tile: Tuple[slice, ...]) -> None:
             # Write group metadata
             with tiledb.Group(output_path, "w") as group:
                 group.meta.update(
-                    reader.group_metadata,
+                    **reader.group_metadata,
```

Comment:
No need for `**`, the first (and only) positional parameter to update can be a mapping.